AcceptanceTests/AcceptanceTests.Common/Driver/Drivers/UserBrowser.cs

```diff
@@ -105,15 +105,16 @@ public void Retry(Action action, int times = ActionRetries)
                 .Execute(action);
         }
 
-        private void RetryOnStaleElement(Action action, int times = ActionRetries)
-        {
-            Policy
-                .Handle<StaleElementReferenceException>()
-                .WaitAndRetry(times, retryAttempt =>
-                        TimeSpan.FromSeconds(Math.Pow(2, retryAttempt)),
-                    (exception, timeSpan, context) => { NUnit.Framework.TestContext.WriteLine($"Encountered error '{exception.Message}' after {timeSpan.Seconds} seconds. Retrying..."); })
-                .Execute(action);
-        }
+        //  09/06/2021 - Keep for now as this might be meeded to be restored.
+        //private void RetryOnStaleElement(Action action, int times = ActionRetries)
+        //{
+        //    Policy
```

Comment:
I personally prefer getting rid of it and taking note of the commit ID. You can then just grab it from history. Comments tend to get left when we do stuff like this.

Reply:
I'm not sure that I like the idea but that's just my preference.  Code has been removed and will be pushed.